src/main/java/Duke.java

```diff
@@ -1,10 +1,113 @@
+import java.util.Scanner;
+
 public class Duke {
+    public static final String TEXT_SPACER = "    ";
+
     public static void main(String[] args) {
         String logo = " ____        _        \n"
                 + "|  _ \\ _   _| | _____ \n"
                 + "| | | | | | | |/ / _ \\\n"
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+
+        printLineSpacer();
+        System.out.println(TEXT_SPACER + "Hey, what's up!\n" + TEXT_SPACER + "What can I help you with today?");
+        printLineSpacer();
+
+        Task[] userLists = new Task[100];
+        Scanner scanner = new Scanner(System.in);
+        String userLineInput = "";
+        int numOfTasks = 0;
+
+        do {
+            userLineInput = scanner.nextLine();
+            String[] splitInputs = userLineInput.split(" ");
+            String userCommand = splitInputs[0];
+            printLineSpacer();
+            if (!userCommand.equals(("bye"))) {
+                switch (userCommand) {
+                case "list":
+                    printList(userLists, numOfTasks);
+                    break;
+                case "done":
+                    completeTask(userLists, splitInputs);
+                    break;
+                case "event":
+                case "deadline":
+                case "todo":
+                    storeTasks(userLists, userLineInput, numOfTasks, userCommand);
+                    numOfTasks++;
+                    break;
+                default:
+                    System.out.println(TEXT_SPACER + "Invalid Input!");
+                    printLineSpacer();
+                    break;
+                }
+            }
+        } while(!userLineInput.equals("bye"));
+
+        System.out.println(TEXT_SPACER + "Aight. See you soon mate.");
+        printLineSpacer();
+    }
+
+    private static void printList(Task[] userLists, int numOfTasks) {
```

Comment:
Great choice to abstract the code used to print out details about each tasks, makes the overall code much neater and easier to read!